src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -573,9 +578,14 @@ class GroupReadsByUmi
       // Count up the family sizes
       templatesByMi.values.foreach(ps => tagFamilySizeCounter.count(ps.size))
     }
-
     out.close()
 
+    logger.info(f"Accepted $kept%,d reads for grouping.")
+    if (filteredNonPf > 0) logger.info(f"Filtered out $filteredNonPf%,d non-PF reads.")
+    logger.info(f"Filtered out $filteredPoorAlignment%,d reads due to mapping issues.")
+    logger.info(f"Filtered out $filteredNsInUmi%,d reads that contained one or more Ns in their UMIs.")
+    this.minUmiLength.foreach { _ => logger.info(f"Filtered out $filterUmisTooShort%,d reads that contained UMIs that were too short.") }
```

Comment:
It makes me a little sad that now if GroupReads is doing the sorting I have to wait until the output is written to get the stats.

Reply:
I have found a way, I think